### PR TITLE
Add link to graphql-mocks for graphql docs

### PR DESCRIPTION
**Pull request by chadian**
@samselikoff @jneurock 
As discussed on discord, this pull request adds a link in the graphql docs to `graphql-mocks` and an alternative Mirage solution for mocking GraphQL APIs.

Just wanted to get something "on the page", happy to make any edits.

### CODE REVIEW

File: src/routes/docs/advanced/graphql.mdx [hunk 1]
```diff
@@ -98,6 +98,16 @@ There are a lot more things Mirage GraphQL can do. For example, you can
 
 and more.
 
-Mirage GraphQL is v0.1.*x*, but extracted from years of work on the `ember-cli-mirage-graphq` library.
+Mirage GraphQL is v0.1.*x*, but extracted from years of work on the `ember-cli-mirage-graphql` library.
```
Comment (jneurock) [inline src/routes/docs/advanced/graphql.mdx:101]: Thanks for fixing my typo!

File: src/routes/docs/advanced/graphql.mdx [hunk 2]
```diff
@@ -98,6 +98,16 @@ There are a lot more things Mirage GraphQL can do. For example, you can
...
 To learn more, check out the README of [Mirage GraphQL](https://github.com/miragejs/graphql). Also check out the intergration tests for more in-depth examples of options, queries and mutations.
+
+## Mirage & `graphql-mocks`
+
+As an unofficial community-based alternative to mocking with Mirage and GraphQL check out [`graphql-mocks`](https://www.graphql-mocks.com) and its [Mirage middleware](https://www.graphql-mocks.com/docs/guides/mirage-js).
+
+Consider using `graphql-mocks` with Mirage for:
+* Composable middlewares for managing and creating different scenarios and states
+* Resolver and resolver map abstractions for extending query resolutions
+* Agnostic network and support for Node
```
Comment (jneurock) [inline src/routes/docs/advanced/graphql.mdx:112]: I don't quite understand this point and I'm not sure how to suggest anything helpful

**Review by jneurock — APPROVED** [review]
LGTM but I wonder if @samselikoff or @ryanto would like to take a look, too?

**Comment by samselikoff** [comment]
Looks great! Sorry this hasn't been as high a priority for me lately but very happy to see you extending things to fit your needs 👍

**Comment by chadian** [comment]
@jneurock @samselikoff Any chance this can be merged?

**Comment by samselikoff** [comment]
sorry about the delay! I think I just forgot to click the button 😆

**Pull request merged**